plugins/content/caption/base/CaptionPlugin.php

```diff
@@ -670,17 +670,21 @@ protected static function getLiveCaptionArray($entryId, $deliveryProfile): array
 		}
 
 		$liveCaptions = array();
+		$baseUrl = $deliveryProfile->getPackagerUrl($liveEntryServerNodes[0]);
+		$url = parse_url($baseUrl, PHP_URL_PATH);
+		$prefix = substr($baseUrl, 0, -strlen($url));
```

Comment:
getting the prefix out of the request is wrong, assuming that future wise we would send a query param like ?test=1 , this code will break